packages/react-composites/src/composites/MeetingComposite/MeetingCallControlBar.tsx

```diff
@@ -23,17 +23,28 @@ export interface MeetingCallControlBarProps {
   onPeopleButtonClicked: () => void;
   mobileView: boolean;
   disableButtonsForLobbyPage: boolean;
+  meetingCallControlOptions?: MeetingCallControlOptions;
 }
 
 /**
  * @private
  */
 export const MeetingCallControlBar = (props: MeetingCallControlBarProps): JSX.Element => {
-  // Disable the default participants button as meetings composite has its own participants button.
+  // Set the desired control buttons from the meetings composite. particiapantsButton is always false since there is the peopleButton.
   let callControlsOptions: CallControlOptions | false = {
     participantsButton: false,
-    screenShareButton: props.mobileView ? false : { disabled: props.disableButtonsForLobbyPage }
+    screenShareButton:
+      props.mobileView || !props.meetingCallControlOptions?.screenShareButton
+        ? false
+        : { disabled: props.disableButtonsForLobbyPage },
+    displayType: props.meetingCallControlOptions?.displayType,
+    microphoneButton: !props.meetingCallControlOptions?.microphoneButton ? false : true,
+    cameraButton: !props.meetingCallControlOptions?.cameraButton ? false : true,
+    devicesButton: !props.meetingCallControlOptions?.devicesButton ? false : true
```

Comment:
Explicit values of `true` and `false` feel very unergonomic. Also, if/when `deviceButton` becomes an object with more internal options, this will need to be unnecessarily updated.

Why is this isn't just `deviceButton: props.meetingCallControlOptions?.deviceButton` ? If there is a reason, please add a comment in the code explaining why. It confused me, might confused the next reader as well.

Reply:
Fix for previous comment also resolves this. was just making sure the correct logical value is passed down to the callcontrols component. but just passing the meetingControls prop down there directly removes the need for these explicit definitions.